arch/arm/boot/dts/overlays/Makefile

```diff
@@ -34,6 +34,7 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	cap1106.dtbo \
 	chipdip-dac.dtbo \
 	cma.dtbo \
+	cutiepi-panel.dtbo \
```

Comment:
The overlay will need an entry in the README file (so users can run `dtoverlay -h cutiepi-panel` to find out what it does).

Reply:
Thanks, added in last commit.